spec.html

```diff
@@ -38578,8 +38578,7 @@ <h1>
           <emu-alg>
             1. Assert: _p_ is a canonical, unaliased Unicode property name listed in the “Canonical property name” column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref>.
             1. Assert: _v_ is a property value or property value alias for the Unicode property _p_ listed in <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a>.
-            1. Let _value_ be the canonical property value of _v_ as given in the “Canonical property value” column of the corresponding row.
-            1. Return the List of Unicode code points _value_.
+            1. Return the “long name” corresponding to the “long name” or “short name” _v_ in <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a>.
```

Comment:
@michaelficarra I had to adjust this for correctness; _v_ might already be a long name here.

Reply:
> Second Field: The second field is the short name for the property value.
> It is typically an abbreviation, but in a number of cases it is simply
> a duplicate of the "long name" in the third field.

This is unambiguously the "short name", even if it's also the "long name", no?

Reply:
The problem I have with the wording now is that it permits a scenario where `v` is a long name *as well as* a short name associated with some other long name (even though that is likely prohibited in the Unicode data file). How about something like this?

```suggestion
            1. If _v_ is a “short name” associated with some “long name” _l_ for <emu-not-ref>property name</emu-not-ref> _p_ in <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a>, return _l_; otherwise, return _v_.
```

Reply:
That doesn't quite work because there may be multiple aliases as in e.g. `/\p{Script=Qaac}/u` (<q><em>sc ; Copt                             ; Coptic                           ; Qaac</em></q> specifies short names "Copt" and "Qaac" for long name "Coptic"), but I see where you're going. Updated.

Reply:
That new wording WFM.